airflow/contrib/operators/s3_to_sftp_operator.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from airflow.models import BaseOperator
+from airflow.hooks.S3_hook import S3Hook
+from airflow.contrib.hooks.ssh_hook import SSHHook
+from tempfile import NamedTemporaryFile
+from urllib.parse import urlparse
+from airflow.utils.decorators import apply_defaults
+
+
+class S3ToSFTPOperator(BaseOperator):
+    """
+    This operator enables the transferring of files from S3 to a SFTP server
+    :param sftp_conn_id:    The sftp connection id. The name or
```

Comment:
Hi @wmorris75 @Fokko , possibly this comment is coming in a bit late, but I would like to remind that an empty line may be needed after line 30 to make sure this docstring can be parsed properly, according to Sphinx standard. The same for the docstring in other files in this PR.

CC @kaxil

Reply:
How do you test these kind of violations? Would it be possible to automatically check this in the CI? Maybe run Sphinx and check the number of errors?

Reply:
I just happened to notice it. A bit sensitive to this since I kept making this error as well previously.

Running Sphinx may not work. There would be no parsing exception raised explicitly. Instead, it would silently "fail" the proper displaying of the first parameter directly. @kaxil kindly helped fix quite many such cases previously (a big 👍)

So may need committers to pay extra attention to docstrings. Will let you know if I find any way to check this in CI later.